binaries/scootapi/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"git.apache.org/thrift.git/lib/go/thrift"
+	"github.com/scootdev/scoot/scootapi/client"
+	"log"
+)
+
+func main() {
+	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
```

Comment:
Probably, because you need the client and server to match. I expect you may end up with different flavors of Scoot using different protocols and transports.